folder.rb

```diff
@@ -0,0 +1,62 @@
+require 'fileutils'
+
+class Folder
+
+  def self.check_folder(folder)
+
+    return false if !File.directory?(folder)
+
+    folders = Dir.entries(folder)
+
+    if check_important(folders).inspect == 'true'
+      return true
+    end
+
+    folders.each do |f|
+      next if f.include?(".")
+      if File.directory?("#{folder}/#{f}")
+        subfolder= folder + "/" + f
+        puts "Found subfolder: " + subfolder
+        return true if check_folder(subfolder).inspect == 'true'
```

Comment:
```suggestion
        return true if check_folder(subfolder)
```

Using `.inspect` doesn't seems necessary here